pandas/core/arrays/categorical.py

```diff
@@ -1203,6 +1219,24 @@ def map(self, mapper):
     __le__ = _cat_compare_op('__le__')
     __ge__ = _cat_compare_op('__ge__')
 
+    __add__ = _cat_arithmetic_op(operator.add)
```

Comment:
That's a good idea.  For the most part I'm happy being hands-off with Categorical since others are more familiar with it.  I figured this would be an easy way to demonstrate what I have in mind for discussions in #21160 and #21191.